furnace.c

```diff
@@ -200,9 +217,12 @@ command_handler(furnace_context_t* ctx, uint8_t* buffer, void (*feedback)(const
       feedback(msg, msg_len);
   } else if (sscanf(buffer, "pwm %u", &arg) == 1) {
     const int res= set_pwm_safe(FURNACE_FIRE_PIN, ctx, arg);
+#if CONFIG_AUTO == CONFIG_AUTO_PILOT || CONFIG_AUTO == CONFIG_AUTO_MAPPER
```

Comment:
Consider this:

```C
if (res == 0) {
#if CONFIG_AUTO == CONFIG_AUTO_PILOT || CONFIG_AUTO == CONFIG_AUTO_MAPPER
  ctx->pilot.is_enabled = 0;
#endif
}
```

Empty if statements will be removed by the optimizer.
If we get a warning - just do
```C
if (res == 0) {
  (void) res;
#if CONFIG_AUTO == CONFIG_AUTO_PILOT || CONFIG_AUTO == CONFIG_AUTO_MAPPER
  ctx->pilot.is_enabled = 0;
#endif
}
```

Reply:
Version without void casting works without warnings, so i included this one.